<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url('png/clouds.jpg');
background-repeat: no-repeat;
background-position: center;
background-size: cover;</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,5,3">
    <property name="topMargin">
     <number>14</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">Background: white;
border-radius: 10px;
 padding: 10px 20px;</string>
      </property>
      <property name="text">
       <string>Ваши задания на сегодня</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scroll_area">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scroll_widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>945</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="users_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: rgb(0, 148, 216);
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
}
QPushButton:hover {
    background: rgb(0, 120, 190); /* Цвет при наведении */
}
QPushButton:pressed {
    background: rgb(0, 100, 160); /* Цвет при нажатии */
}</string>
        </property>
        <property name="text">
         <string>Участники</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tasks_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: rgb(0, 148, 216);
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
}
QPushButton:hover {
    background: rgb(0, 120, 190); /* Цвет при наведении */
}
QPushButton:pressed {
    background: rgb(0, 100, 160); /* Цвет при нажатии */
}</string>
        </property>
        <property name="text">
         <string>Задания</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="history_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: rgb(0, 148, 216);
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
}
QPushButton:hover {
    background: rgb(0, 120, 190); /* Цвет при наведении */
}
QPushButton:pressed {
    background: rgb(0, 100, 160); /* Цвет при нажатии */
}</string>
        </property>
        <property name="text">
         <string>История</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: rgb(0, 148, 216);
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
}
QPushButton:hover {
    background: rgb(0, 120, 190); /* Цвет при наведении */
}
QPushButton:pressed {
    background: rgb(0, 100, 160); /* Цвет при нажатии */
}</string>
        </property>
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="leave_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: rgb(0, 148, 216);
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px 20px;
}
QPushButton:hover {
    background: rgb(0, 120, 190); /* Цвет при наведении */
}
QPushButton:pressed {
    background: rgb(0, 100, 160); /* Цвет при нажатии */
}</string>
        </property>
        <property name="text">
         <string>Покинуть блок</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
